### PR TITLE
Updating references to Drupal 10 and adding EOL warnings

### PR DESCRIPTION
**Pull request by camikazegreen**
Bringing the homepage of digital.arizona.edu up to date with the various changes made to Quickstart and Bootstrap.

### CODE REVIEW

**Comment by trackleft** [comment]
Preview link: https://refined-github-html-preview.kidonng.workers.dev/az-digital/digital.arizona.edu/raw/task/45/docs/index.html

**Review by joeparsons — APPROVED** [review]
Looks good to me.  Thanks.

File: docs/index.html [hunk 1]
```diff
@@ -53,14 +53,14 @@ <h3 class="card-title text-blue d-inline-block">Arizona Bootstrap</h3>
...
-                <span class="badge badge-success">Drupal 9</span>
+                <h3 class="card-title text-blue d-inline-block">Quickstart 2</h3>
+                <span class="badge badge-success">Drupal 10</span>
                 <span class="badge badge-blue">New</span>
-                <p>A new version of the University's preconfigured version of Drupal is here! Quickstart 2.0 (AZ Quickstart) leverages the power of Drupal 9 combined with Arizona Bootstrap to provide a foundation for developers building Drupal-based sites.</p>
-                <p>Quickstart 1.0 developers and site owners can expect an upgrade path from Quickstart 1.0 to Quickstart 2.0.</p>
+                <p>A new version of the University's preconfigured version of Drupal is here! Quickstart 2 (AZ Quickstart) leverages the power of Drupal 10 combined with Arizona Bootstrap to provide a foundation for developers building Drupal-based sites.</p>
+                <p>The <a href="https://github.com/az-digital/az_quickstart/tree/main/modules/custom/az_migration">az_migrations module</a> provides an upgrade path from Quickstart 1.0 to Quickstart 2.</p>
```
Comment (bberndt-uaz) [inline docs/index.html:60]:
```suggestion
                <p>The <a href="https://github.com/az-digital/az_quickstart/tree/main/modules/custom/az_migration">az_migration module</a> provides an upgrade path from Quickstart 1.0 to Quickstart 2.</p>
```

**Review by bberndt-uaz — APPROVED** [review]

File: docs/index.html [hunk 2]
```diff
@@ -53,14 +53,14 @@ <h3 class="card-title text-blue d-inline-block">Arizona Bootstrap</h3>
 
             <div class="card">
               <div class="card-body">
-                <h3 class="card-title text-blue d-inline-block">Quickstart 2.0</h3>
-                <span class="badge badge-success">Drupal 9</span>
+                <h3 class="card-title text-blue d-inline-block">Quickstart 2</h3>
+                <span class="badge badge-success">Drupal 10</span>
                 <span class="badge badge-blue">New</span>
```
Comment (camikazegreen) [inline docs/index.html:58]:
```suggestion
```

File: docs/index.html [hunk 3]
```diff
@@ -43,7 +43,7 @@ <h2>Latest Versions</h2>
                 <h3 class="card-title text-blue d-inline-block">Arizona Bootstrap</h3>
                 <span class="badge badge-success">Bootstrap 4</span>
                 <span class="badge badge-blue">New</span>
```
Comment (camikazegreen) [inline docs/index.html:45]:
```suggestion
```

File: docs/index.html [hunk 4]
```diff
@@ -75,6 +75,10 @@ <h2>Legacy Versions</h2>
                 <h3 class="card-title text-blue d-inline-block">UA Bootstrap</h3>
                 <span class="badge badge-success">Bootstrap 3</span>
                 <span class="badge badge-warning">Legacy</span>
+                <div class="callout callout-red">
+                  <p class="h6">End of Life Warning</p>
```
Comment (camikazegreen) [inline docs/index.html:78]:
```suggestion
                  <p class="h6 mt-0">End of Life Warning</p>
```

**Review by bberndt-uaz — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Review by danahertzberg — APPROVED** [review]

**Pull request merged**